pennylane/ops/qubit/observables.py

```diff
@@ -50,13 +50,13 @@ class Hermitian(Observable):
     * Gradient recipe: None
 
     Args:
-        A (array or Sequence): square hermitian matrix
-        wires (Sequence[int] or int): the wire(s) the operation acts on
+        A (array or Sequence): square hermitian matrix.
+        wires (Sequence[int] or int): the wire(s) the operation acts on.
```

Comment:
This is not strictly required according to our [style guide](https://docs.pennylane.ai/en/stable/development/guide/documentation.html#punctuation-and-spelling).

Reply:
My bad!